app/ts/types/wire-types.ts

```diff
@@ -9,6 +9,7 @@ const BigIntParser: funtypes.ParsedValue<funtypes.String, bigint>['config'] = {
 	},
 	serialize: value => {
 		if (typeof value !== 'bigint') return { success: false, message: `${typeof value} is not a bigint.`}
+		if (value < 0n) return { success: false, message: `${typeof value} is not a non negative bigint.`}
```

Comment:
I know that in the primary consensus protocol there are no negative numbers, but are there really no negative numbers anywhere in the Ethereum JSON-RPC?